scripts/zones/West_Ronfaure/npcs/Vilatroire.lua

```diff
@@ -5,29 +5,200 @@
 -- "Advanced Teamwork"
 -- !pos -260.361 -70.999 423.420 100
 -----------------------------------
+local ID = require("scripts/zones/West_Ronfaure/IDs")
 require("scripts/globals/quests")
+require("scripts/globals/titles")
+require("scripts/globals/status")
+require("scripts/globals/npc_util")
 -----------------------------------
 
 function onTrade(player, npc, trade)
 end
 
 function onTrigger(player, npc)
+    --player:startEvent(129) -- starts the ready check for all three quests
+    --player:startEvent(130) -- post third quest dialog
     --player:startEvent(131) -- Same job
+    --player:startEvent(132) -- You don't have the requirements to start the second quest
     --player:startEvent(133) -- Same race
```

Comment:
If these events are all implemented into the script now, we no longer need the comments describing them as separate lines~! 😉 

(But do keep the comments describing them when you start them later in the file 😄 )